enclave/key.c

```diff
@@ -269,9 +282,11 @@ oe_result_t oe_public_key_write_pem(
         OE_RAISE(OE_INVALID_PARAMETER);
```

Comment:
Need to print a message here

Reply:
OE_RAISE already print error message by default